### PR TITLE
Update issue links to be more inline with current issue templates

### PR DESCRIPTION
**Pull request by Mm2PL**
Removes link to creating a feature suggestion.

### CODE REVIEW

File: docs/Help.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 Having issues with Chatterino or need general assistance? Check the below troubleshooting below corresponding to your operating system for assistance!
 
-[Report a bug/issue](https://github.com/Chatterino/chatterino2/issues/new?assignees=&labels=bug%2C+needs+triage&template=bug_report.md&title=) | [Make a feature suggestion](https://github.com/Chatterino/chatterino2/issues/new?assignees=&labels=enhancement%2C+needs+triage&template=feature-suggestion.md&title=) | [Join the Discord](https://discord.gg/qq7DDxjste)
+Please search for your problem in the [GitHub Issues][githubissues].  If there
+is no such issue, please create a new one [here][newissue]. If you have more
```
Comment (zneix) [inline docs/Help.md:4]:
```suggestion
Please search for your problem in the [GitHub Issues][githubissues].  If there
is no such issue, please [create a new one][newissue]. If you have more
```

File: docs/Help.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+questions you can [join the Discord](https://discord.gg/qq7DDxjste).
```
[the remark this answers is not in the document]
Comment (ALazyMeme) [reply docs/Help.md:5]: There should be a comma there

[the remark this answers is not in the document]
Comment (Mm2PL) [reply docs/Help.md:5]:
```suggestion
questions, feel free to [join the Discord](https://discord.gg/qq7DDxjste).
```

**Review by xHeaveny — APPROVED** [review]
looks good

**Review by pajlada — APPROVED** [review]

**Pull request merged**